tools/database/README.md

```diff
@@ -62,6 +62,18 @@ Then you can call
 `neard database change-db-kind --new-kind Cold change-hot`.
 Notice that even though in your mind this db is cold, in your config this db hot, so you have to pass `change-hot`.
 
+## Compact database
+
+Run compaction on the SST files. After running this the performance of database will increase significantly.
```

Comment:
suggestion: `After running this the performance of database will increase significantly.` -> `Running this command might increase read performance.`